### ISSUE TITLE
Investigate changes to game data APIs

### ISSUE THREAD

**Issue by danjagnow** [blizzard-net/warcraft#40]
Review the [World of Warcraft Game Data APIs Preliminary Release](https://us.battle.net/forums/en/bnet/topic/20757557566) post on Blizzard's API Discussion forum and create issues for any required changes to the client library.

**Comment by TravisBoatman** [comment]
Looks like the biggest changes are in localization and authorization. 

If you don't give a locale the result will look like this...
```
        "name": {
            "en_US": "North America",
            "es_MX": "Norteamérica",
            "pt_BR": "América do Norte",
            "de_DE": "Nordamerika",
            "en_GB": "North America",
            "es_ES": "Norteamérica",
            "fr_FR": "Amérique du Nord",
            "it_IT": "Nord America",
            "ru_RU": "Северная Америка",
            "ko_KR": "미국",
            "zh_TW": "北美",
            "zh_CN": "北美"
        }
```

If you do give a locale it will display like normal...
```
"name": "North America"
```

As for the authorization we'll need to obtain an `access_token` first before we can make any requests. We can't just provide a `client_id` query parameter anymore. See how to [here](https://dev.battle.net/docs/read/oauth#ClientCredentialsFlow).

**Comment by danjagnow** [comment]
Blizzard's migration to the new Game Data and Profile Data APIs is almost complete.  The migration status is addressed in #70.
